modules/javafx.graphics/src/main/java/com/sun/prism/impl/Disposer.java

```diff
@@ -126,6 +128,10 @@ public static void disposeRecord(Disposer.Record rec) {
      * thread on which  the resources were created).
      */
     public static void cleanUp() {
+        if (!Thread.currentThread().getName().startsWith("QuantumRenderer")) {
```

Comment:
This seemed like a dubious way to detect the render thread so I looked around for how it is done elsewhere but instead I found
com/sun/javafx/sg/prism/NGCanvas.java
    private static void runOnRenderThread(final Runnable r) {
        // We really need a standard mechanism to detect the render thread !
        if (Thread.currentThread().getName().startsWith("QuantumRenderer")) {

And .. it is also used by printing, and I added it in 2013 :-(

In my defence I copied the code from webview as mentioned in https://bugs.openjdk.org/browse/JDK-8094524

oh well.